learn/app-development/deployment/build-with-docker.md

```diff
@@ -63,7 +63,7 @@ Save the file to the project's home directory. The Docker automatically picks up
 Build a Docker image by using the below Docker command with different build profiles. You can choose to build with the following build profiles including `development` and `deployment`.
 
 ```Docker
-docker image build --build-arg build_profile_name=<deployment-profile> -t <imagename:version> <Dockerfile_location>
```

Comment:
Save the file to the project's home directory.  ==>>>
Save the Dockerfile.build to the project's home directory. So that Docker build will pick this.